frontend/packages/ceph-storage-plugin/src/components/create-storage-system/payloads.ts

```diff
@@ -46,7 +46,7 @@ export const createNoobaaPayload = (): Payload => {
       plural,
     },
     payload: {
-      apiVersion,
+      apiVersion: 'noobaa.io/v1alpha1',
```

Comment:
extract as const and remove hardcoding

Reply:
There are a lot of properties in this payload. I would need to change all of them then.
It does not makes sense to make it `const` here.

Reply:
hmm..... i don't get it. What other properties are we talking about ? We already do this in many places. We have done it for storage system in this same file
Sorry for misunderstanding, what i meant was using `apiVersionForModel(NooBaaSystemModel)` instead of hardcode.
that should work right ?

Reply:
That I can do. If you make it const then it does not makes sense